app/mqtt_publish_automations.py

```diff
@@ -97,11 +98,10 @@ def get_automation_url(
 
 def get_relative_path(deployment_id: str, bs_yaml: dict | None) -> str | None:
     """Get relative path from bitswan.yaml"""
-    if not bs_yaml or "deployments" not in bs_yaml:
```

Comment:
This change will cause it to raise a KeyError if "deployments" section is not present. Is this desired?